Library/Homebrew/startup/config.rb

```diff
@@ -42,6 +42,9 @@
 # Where build, postinstall, and test logs of formulae are written to
 HOMEBREW_LOGS = Pathname(ENV.fetch("HOMEBREW_LOGS")).expand_path.freeze
 
+# Path to the list of Homebrew maintainers as a JSON file
+HOMEBREW_MAINTAINER_JSON = (HOMEBREW_REPOSITORY/"maintainers.json").freeze
```

Comment:
Not really sure about putting this here, but I figured I should put it somewhere so it can be used in `test-bot`.

Reply:
I just put it in `.github` since we already have `actionlint-matcher.json` there. Happy to move it deeper if you prefer.